src/lib/network/wicked_handlers.rb

```diff
@@ -0,0 +1,20 @@
+# encoding: utf-8
+
+require "yast"
+
+module Yast
+  module Wicked
```

Comment:
well, I tried to find a ruby style guide about this and found nothing so far.

And in fact e.g. module name ```WickedHandlers``` vs file name ```wicked_handlers``` is not much better than this one.